grails-app/migrations/0.9.x/changelog-2024-10-04-1610-assign-amount-to-invoice-items.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="anadolny" id="041020241610-0">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <columnExists tableName="invoice_item" columnName="amount"/>
+                <not>
```

Comment:
Weird that liquibase requires this clunky syntax and doesn't have a better solution like `expectedResult=">0"` or `<sqlCheck notExpectedResult="0">`